app/views/clients/index.html.erb

```diff
@@ -14,18 +14,18 @@
     <tbody>
       <% @clients.each do |client| %>
         <tr>
-          <td><%= client.first_name %></td>
-          <td><%= client.last_name %></td>
+          <td><%= client.first_name.humanize %></td>
+          <td><%= client.last_name.humanize %></td>
```

Comment:
I guess instead of first and last name in a separate column, use one `name` method in one `td`